client/src/components/Recomendations/Recomendations.tsx

```
@@ -0,0 +1,63 @@
+        fetchSuggestions();
+    }, []);
+
+    return (
+        <div>
+            {loading && <p>Loading...</p>}
+            {error && <p>{error}</p>}
+            {suggestions.length > 0 && (
+                <ul className='flex items-center justify-center'>
+                    <Marquee pauseOnHover vertical className="[--duration:20s]">
+                            {suggestions.map((suggestion, index) => (
+                                <div className="flex items-center border border-gray-300 rounded-lg">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)